### PR TITLE
Implement prototype v128.load{32,64}_zero instructions

### PR DESCRIPTION
**Pull request by tlively**
Specified in https://github.com/WebAssembly/simd/pull/237. Since these
are just prototypes necessary for benchmarking, this PR does not add
support for these instructions to the fuzzer or the C or JS APIs. This
PR also renumbers the QFMA instructions that previously used the
opcodes for these new instructions. The renumbering matches the
renumbering in V8 and LLVM.

### CODE REVIEW

File: test/simd.wast.fromBinary [hunk 1]
```diff
@@ -896,8 +896,10 @@
   )
  )
  (func $f32x4.qfma (param $0 v128) (param $1 v128) (param $2 v128) (result v128)
-  (f32x4.qfma
+  (drop
```
Comment (aheejin) [inline test/simd.wast.fromBinary:899]: Is this correct change?

File: src/wasm-binary.h [hunk 2]
```diff
@@ -888,8 +888,11 @@ enum ASTNodes {
   F32x4ConvertSI32x4 = 0xfa,
   F32x4ConvertUI32x4 = 0xfb,
 
-  F32x4QFMA = 0xfc,
-  F32x4QFMS = 0xfd,
+  V128Load32Zero = 0xfc,
+  V128Load64Zero = 0xfd,
+
+  F32x4QFMA = 0xb4,
```
Comment (MaxGraey) [inline src/wasm-binary.h:894]: `0xb4` already uses for `I32x4DotSVecI16x8`:
https://github.com/WebAssembly/binaryen/blob/master/src/wasm-binary.h#L836

Is it correct?

**Comment by tlively** [comment]
Yes, this is the correct renumbering, although the opcode collision in unintentional. I will shift the dot instruction opcode to use the number used by V8 as well. See https://github.com/WebAssembly/tool-conventions/blob/master/Linking.md#target-features-section for the numbering I am following.

**Comment by MaxGraey** [comment]
It seems it also require some test suite changes. When I try run `./auto_update_tests.py` script I got:
```
[parse exception: v128.load32_zero (at 1148:2)]
Fatal: error parsing wasm
Traceback (most recent call last):
  File "./auto_update_tests.py", line 260, in <module>
    sys.exit(main())
  File "./auto_update_tests.py", line 254, in main
    TEST_SUITES[test]()
  File "/Volumes/Archive/Projects/Github/binaryen/scripts/test/wasm_opt.py", line 187, in update_wasm_opt_tests
    actual = support.run_command(cmd)
  File "/Volumes/Archive/Projects/Github/binaryen/scripts/test/support.py", line 176, in run_command
    raise Exception(('run_command failed (%s)' % code, out + str(err or '')))
Exception: ('run_command failed (1)', '')

```

**Pull request merged**